src/vault.py

```diff
@@ -0,0 +1,41 @@
+from fabric.api import task, local
+from buildercore import project
+import utils
+import sys
+import logging
+LOG = logging.getLogger(__name__)
+
+def vault_addr():
+    defaults, _ = project.raw_project_map()
+    return defaults['aws']['vault']['address']
+
+@task
+def login():
+    cmd = "VAULT_ADDR=%s vault login" % vault_addr()
+    local(cmd)
+
+@task
+def logout():
+    cmd = "rm -f ~/.vault-token"
+    local(cmd)
+
+@task
+def lookup(token):
+    cmd = "VAULT_ADDR=%s VAULT_TOKEN=%s vault token lookup" % (vault_addr(), token)
+    local(cmd)
+
+@task
+def create_token():
+    fname = utils.get_input('first name: ')
```

Comment:
a display name seems to have no preferred structure, so I just stuck to our NameSurname convention but this could be used also for applications e.g. `bot` or `search` or `journal`. I'd leave a single custom value as the input.

I'd also allow for the parameter to be passed in as a non-interactive argument, to remain automation-friendly.